package.json

```diff
@@ -25,6 +25,8 @@
   },
   "homepage": "https://github.com/lob/hapi-rate-limit",
   "dependencies": {
+    "redis": "^0.12.1",
```

Comment:
Can we use a newer version of this package? Looks like its up to v2.6 now: https://github.com/NodeRedis/node_redis/blob/master/package.json